Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM node:8.1.4
+FROM node:8.9.0-alpine
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 COPY package.json yarn.lock /usr/src/app/
```

Comment:
I left this a separate step to speed up build time. If copying source files happened before `yarn install` it would mean that every tiniest file change would trigger full dependencies installation.

Unfortunately that increases image size to around ~500MB because yarn cache remains as separate layer.